woocommerce/woocommerce
# Revert low stock notification changes

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR reverts the commits, 623b1f3, 10bc399, ad94888 that are related to product low stock notifications

### Testing instructions:

Scenario A - duplicating product.
- Have an email loggin extension installed.
- Create a product, with tracked inventory, with stock as 0.
- Now duplicate the product.
- You should see an low stock email received in the email logger on `trunk` and not in this branch.

Scenario B - Bulk edit
- In the product list page
- Select any product and click edit in the bulk edit 
- Click update without changing anything
- You should see an low stock email received in the email logger on `trunk` and not in this branch.

Scenario C - Bulk import
- Bulk import the products with 0 quanity.
- You should see a bunch of low stock email received in the email logger on `trunk` and not in this branch.
